dapp/client/App.elm

```diff
@@ -67,19 +73,33 @@ update : Message -> Model -> ( Model, Cmd Message )
 update message model =
     case message of
         AddressChanged newAddress ->
-            ( { model | address = Just newAddress }, Cmd.none )
+            ( { initialModel | address = Just newAddress }, Cmd.none )
```

Comment:
The `{ x | a = b}` syntax means, "start with object/model `x` and then set attribute/property `a` on it to `b`" (all immutably of course, without actually changing `x`)?